<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnTech::GuiQt::Entity::EntityFunctionTablesWidget</class>
 <widget class="QWidget" name="UnTech::GuiQt::Entity::EntityFunctionTablesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entity ROM Structs Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="PropertyTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QToolBar" name="tableButtons">
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="floatable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PropertyTableView</class>
   <extends>QTreeView</extends>
   <header>gui-qt/common/properties/propertytableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
